modules/ui/background.js

```diff
@@ -318,6 +354,39 @@ export function uiBackground(context) {
         }
     }
 
+    function getBackgrounds(filter) {
+        return context.background()
+            .sources(context.map().extent(), context.map().zoom(), true)
+            .filter(filter);
+    }
+
```

Comment:
Yet another formatting nit- functions in iD have two spaces between them. NOT a big deal, but something to note!